src/helpers.sh

```diff
@@ -153,3 +153,36 @@ function helper::trim() {
 
     echo "$trimmed_string"
 }
+
+function helpers::get_latest_tag() {
+    local remote_url="https://github.com/TypedDevs/bashunit"
+    local latest_tag
+
+    latest_tag=$(git ls-remote --tags "$remote_url" | awk '{print $2}' |
+    sed 's|^refs/tags/||' | sort -V | tail -n 1
+    )
+
+    echo "$latest_tag"
```

Comment:
Would this work out as well, without the need for a local variable?
```suggestion
  git ls-remote --tags "$remote_url" |
    awk '{print $2}' |
    sed 's|^refs/tags/||' |
    sort -V |
    tail -n 1
```

Reply:
changed it